.github/workflows/ci.yml

```diff
@@ -206,7 +206,7 @@ jobs:
         run: |
           set -euxo pipefail
           wget -qO- https://apt.llvm.org/llvm-snapshot.gpg.key | sudo tee /etc/apt/trusted.gpg.d/apt.llvm.org.asc
-          echo deb http://apt.llvm.org/jammy/ llvm-toolchain-jammy main | sudo tee /etc/apt/sources.list.d/llvm.list
+          echo deb http://apt.llvm.org/jammy/ llvm-toolchain-jammy-18 main | sudo tee /etc/apt/sources.list.d/llvm.list
           sudo apt update
           sudo apt -y install clang gcc-multilib llvm locate qemu-system-{arm,x86}
```

Comment:
You need to update this to install `llvm-18`. The CI log contains
```
Selecting previously unselected package llvm.
Preparing to unpack .../091-llvm_1%3a14.0-55~exp2_amd64.deb ...
Unpacking llvm (1:14.0-55~exp2) ...
```
because `llvm` is now being provided from the default ubuntu repos.

You'll probably have to do additional plumbing to get llvm-sys to find llvm since all the names will be suffixed.